language.md

```diff
@@ -434,6 +434,11 @@ dates, the lesser value occurs temporally earlier.
 Check whether _expr_, which must be a string, must matches the provided regular
 expression.
 
+- _expr_` !~ /`_re_`/`
+
+Check whether _expr_, which must be a string, must not matches the provided
```

Comment:
Perhaps `Check whether...must not matches...` -> ` Check that...does not match...`?

Reply:
Oh, this was some copy-pasta. The other one is also bad.